src/pegout/components/PegoutForm.vue

```diff
@@ -442,6 +470,8 @@ export default defineComponent({
 
     watch(account, clearState);
 
+    getAvailableLiquidity();
```

Comment:
Don't need to call it if flyover is not enabled or LP doesn't exist.

Reply:
In those cases it shouldn't be any LP in the store so the action just resolves, anyways I'll add an extra check just in case.